joomla/joomla-cms
# Normalize some parameters in com_contact and com_content xml files

### Summary of Changes
Normalize some parameters in com_contact and com_content xml files (some parameters was missing in some xml, see below).
Reorder parameters as they shown in views.
Code style (https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md).

`/administrator/components/com_contact/config.xml` - added `show_category_heading_title_text` parameter, reordered parameters of `category`, `categories`, `list_default_parameters` fieldsets, code style.
`/components/com_contact/views/categories/tmpl/default.xml` - added `show_category_heading_title_text`, `show_image_heading`, `initial_sort` parameters, reordered parameters of `basic`, `category`, `advanced` fieldsets, code style.
`/components/com_contact/views/category/tmpl/default.xml` - added `show_category_heading_title_text` parameter, reordered parameters of `category` fieldset, code style.

`/administrator/components/com_content/config.xml` - fixed double names of spacers, reordered parameters of `category`, `categories`, `list_default_parameters` fieldsets, code style.
`/components/com_content/views/categories/tmpl/default.xml` - added `show_category_heading_title_text`, `show_cat_tags`, `page_subheading` parameters, reordered parameters of `basic`, `category`, `advanced` fieldsets, code style.
`/components/com_content/views/category/tmpl/blog.xml` - reordered parameters of `category` fieldset, code style.
`/components/com_content/views/category/tmpl/default.xml` - removed unnecessary `>`, reordered parameters of `category` fieldset, code style.

### Testing Instructions

1. Use a fresh Joomla install with sample items.
2. Create a tree of some contact categories.
3. Create contacts with default options and place they to some contact categories.
4. Create menu items of the types `Contacts » List All Contact Categories` and `Contacts » List Contacts in a Category` for the root contact category.
5. Check the contact categories in the frontend. All should be fine.
6. Play with option 'Show Subcategories Text' in `Global Configuration » Contacts`, in the Category tab of the menu items you created and check everything is fine.

The same for content (menutypes:  `Articles » List All Categories`, `Articles » Category List`, `Articles » Category Blog`)
